docs/analysis-workflows/dna-open-access-filtering.md

```diff
@@ -1,8 +1,47 @@
 ---
 id: dna-open-access-filtering
-title: Open Access Filtering
-sidebar_label: Open Access Filtering
+title: Open Access Variant Filtering
+sidebar_label: Open Access Variant Filtering
 platform_key: DOCS_DNA_PIPELINE
 ---
 
-![Reminder Banner](/assets/submission/banner-reminder.svg)
+The SNV/InDel variant calls from both Sanger & MuTect2 calling results on ARGO portal are further processed through the Open Access Variant Filtering Workflow to create VCF files that can be downloaded by ordinary users from the open-access tier. For details about the process workflow, please see the latest version of the [ARGO Open Access Variant Filtering Workflow](https://github.com/icgc-argo-workflows/open-access-variant-filtering/releases)
+
+## Background
+
+Although in cancer research it is not thought to be possible to re-identify patients from somatic variants data as these mutations are found in the tumour not in an individual's germline genome. That means somatic variants do not provide identifying information theoretically and barrier-free release of somatic variants data can occur without compromising patient privacy.
+
+However germline variants can be mistakenly identified as somatic ones, these are so called "gerline leaks" because:
```

Comment:
change to:
However, germline variants can be mistakenly identified as somatic variants, these are referred to as "germline leaks" because: